gencred/merge_kubeconfig_secret.py

```diff
@@ -34,7 +34,13 @@
 import tempfile
 import time
 
+# Import the Secret Manager client library.
+# Requires `pip install google-cloud-secret-manager`
+from google.cloud import secretmanager
+
 reAutoKey = re.compile('^config-(\\d{8})$')
+gcp_secret_key = "prowkubeconfigbackup"
```

Comment:
nit: Should this be ALL_CAPS?

Reply:
Done, to be more careful maybe they should be `_` prefixed, but I'm not imagining anyone importing this file, so let's keep them as they are